### DISCUSSION TITLE
How to use ui-kitten with the application generated with ignite-cli

### DISCUSSION THREAD

**Discussion by conceptiveconsulting** [infinitered/ignite#1774] [Q&A] [answered]
Hello 

Was wondering how to integrate ui-kitten components with ignite-cli generated application. Im a bit new to the ignite-cli and react-native , any help appreciated. 

Bhushan

**Comment by jamonholmgren** [comment, marked as answer]
@conceptiveconsulting Apologies, I keep forgetting to check Github Discussions!

We've used UI Kitten with a new Ignite app and it worked well. Generally speaking, you can just follow their normal instructions to install and use it: https://akveo.github.io/react-native-ui-kitten/docs/guides/getting-started (go to the "manual installation" instructions partway down).

It would be fun to do this on-stream sometime. I've been streaming coding on Twitch: https://twitch.tv/jamonholmgren

**Comment by conceptiveconsulting** [comment]
Hi Jamon

Thanks for the email. Will connect if we have any issues

Bhushan Ivatury
Founder & CEO
Conceptive Consulting

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
